admin/access-control/organizations.md

```diff
@@ -312,13 +312,23 @@ organization.
 
 ## Namespaces
 
-Coder's helm chart includes `namespaceWhitelist`. This value takes a list of the
-namespaces that are in your cluster and are available to Coder. If you don't set
-`namespaceWhitelist`, the only available namespace is the one Coder has deployed
-to (this is also the default namespace).
+NOTICE: The `namespaceWhitelist` field has been deprecated in Coder version 
+1.17.
 
-If you're a site manager, you can assign a namespace to an organization; all
-resources for that organization now reside in that namespace. **Please note that
-this field is not editable once set.**
+Coder's helm chart previously included a `namespaceWhitelist` field that has 
+been removed in version 1.17. This value took a list of the namespaces that are 
+in your cluster and are available to Coder. 
```

Comment:
If we add the notice to the top, I think we can keep the old wording and just use parentheses with a bold statement like

```md
- (**deprecated in 1.17**) If you're a site manager, you can assign...
```

Reply:
Could you give a full example of the format you are thinking? I'm having a hard time figuring out where I would add that back cleanly.

Reply:
I've pushed a change with the notice header

Reply:
It looks good - all good